x/supplier/client/cli/tx_stake_supplier.go

```diff
@@ -17,15 +20,19 @@ var _ = strconv.Itoa(0)
 func CmdStakeSupplier() *cobra.Command {
 	// fromAddress & signature is retrieved via `flags.FlagFrom` in the `clientCtx`
 	cmd := &cobra.Command{
-		Use:   "stake-supplier [amount]",
+		// TODO_HACK: For now we are only specifying the service IDs as a list of of strings separated by commas.
+		// This needs to be expand to specify the full SupplierServiceConfig. Furthermore, providing a flag to
+		// a file where SupplierServiceConfig specifying full service configurations in the CLI by providing a flag that accepts a JSON string
+		Use:   "stake-supplier [amount] [svcId1;url1,svcId2;url2,...,svcIdN;urlN]",
 		Short: "Stake a supplier",
 		Long: `Stake an supplier with the provided parameters. This is a broadcast operation that
 will stake the tokens and associate them with the supplier specified by the 'from' address.
```

Comment:
NIT: Until we pass in the service config file can you give a brief explainer of the comma separated format and the ; delim usage within it

Reply:
Updated long usage with

```
TODO_HACK: Until proper service configuration files are supported, suppliers must specify the services as a single string
of comma separated values of the form 'service;url' where 'service' is the service ID and 'url' is the service URL.
For example, an application that stakes for 'anvil' could be matched with a supplier staking for 'anvil;http://anvil:8547'.
```